README.md

```diff
@@ -1,3 +1,55 @@
 # @api3/services
 
-> A package that provides information about services already deployed like beacons, airnodes and contract addresses
+> A package that provides information about services already deployed like beacons, airnodes and contract addresses.
+> Also provides tools necessary to create projects which use these services.
+
+## User documentation
+
+The user facing documentation will be added to [API3 docs](https://docs.api3.org/) shorty.
+
+## Developer documentation
+
+This package consists of two parts, both related to beacons:
+
+1. API - to get the data about api3 managed services and a beacon whitelisting function
+2. CLI - a CLI tool, called `create-beacon-reader-app` which creates a starter project for applications using beacon
+   values.
+
+### Installation, building and testing
+
+To install dependencies, run:
+
+```
+yarn
```

Comment:
I didn't know you can just run `yarn` without the `install` [command](https://yarnpkg.com/cli/install) to install all deps

Reply:
yep, you can, but I'll make it say `yarn install` to be more explicit.